doc/classes/SkeletonIK.xml

```diff
@@ -12,12 +12,14 @@
 			<return type="Skeleton">
 			</return>
 			<description>
+                Returns the parent [Skeleton] Node that was present when SkeletonIK entered the [SceneTree]. Returns null if the parent node was not a [Skeleton] Node when SkeletonIK entered the [SceneTree].
```

Comment:
The descriptions should be indented with tabs, not spaces.

Reply:
```suggestion
                Returns the parent [Skeleton] Node that was present when SkeletonIK entered the [SceneTree]. Returns [code]null[/code] if the parent node was not a [Skeleton] Node when SkeletonIK entered the [SceneTree].
```

I'm not sure about the use of past tense though. What happens if the SkeletonIK has not entered the scene tree yet? Or can the parent skeleton be changed later on by reparenting?

Reply:
That is the current quirk, the SkeletonIK function returns a stored variable instead of getting the parent. It only gets the parent node ones on enter_tree notification. Every change later gets ignored.

Reply:
> 
> 
> The descriptions should be indented with tabs, not spaces.

My bad, will change, my IDE was still configured to replace tabs with 4 spaces from something else.